content/docs/latest/spire-helm-charts-hardened-about/identifiers.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Identifiers
+short: Identifiers
+description: How to configure SPIFFE IDs
+kind: spire-helm-charts-hardened-about
+weight: 200
+aliases:
+    - /docs/latest/helm-charts-hardened/identifiers
+---
+
+## Defaults
+
+- The chart deploys [SPIRE Controller Manager](https://github.com/spiffe/spire-controller-manager) to manage SPIFFE Identifiers by Kubernetes Custom Resources.
+- The chart deploys a ClusterSPIFFEID Custom Resource that gives an identifier to all pods of the form `spiffe://{{ .TrustDomain }}/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}`.
+- For a lot of use cases you don't need to add additional identifiers.
+
+## Custom / Additional Identifiers
+
+In some cases you may want to customize your identifiers or add additional ones.
+
+The chart supports managing the Custom Resources from the charts values file for even easier management. We recommend using values for customization.
+
+While you can manage identities using the [Kubernetes Custom Resources](https://github.com/spiffe/spire-controller-manager?tab=readme-ov-file#custom-resources) directly,
+we do not recommend doing that as it takes care to not misconfigure.
+
+## Restricting the default SVIDs.
+
+Some workloads only reliably support one SVID at a time. To support customization you can do either two things:
+
+1. You can disable the default ClusterSVID entirely and load in individual ClusterSVIDs for each workload/namespace.
```

Comment:
You mean ClusterSPIFFEId? There are several references to SVID and ClusterSVID on this page but I don't see the term being introduced.

Reply:
Hmm... good catch. SVID being used here is the SPIFFE/SPIRE definition, so should be assumed here to be known?

The ClusterSVID part though was intended to be ClusterSPIFFEID though. Will fix that one for sure. Let me know on the SVID part.

Reply:
Updated. Please have a look.